examples/ensemble/plot_feature_transformation.py

```diff
@@ -51,13 +51,13 @@
                                                             test_size=0.5)
 
 # Unsupervised transformation based on totally random trees
-rt = RandomTreesEmbedding(max_depth=3, n_estimators=n_estimator)
-rt_lm = LogisticRegression()
-rt.fit(X_train, y_train)
-rt_lm.fit(SelectFromModel(rt, prefit=True).transform(X_train_lr), y_train_lr)
+rt = RandomTreesEmbedding(max_depth=3, n_estimators=n_estimator,
+	random_state=0)
```

Comment:
@yanlend : are these changes supposed to be in your PR? They look strange to me and somewhat unrelated.